sinonjs/sinon
# Improve documentation: async calls (#1286)

#### Purpose (TL;DR) - mandatory
> This is to improve documentation of async stub calls - http://sinonjs.org/releases/v4.1.5/stubs/#stubcallsargasyncindex

#### Background (Problem in detail)  - optional
> Issue: https://github.com/sinonjs/sinon/issues/1286.
> It also fixes duplicated docs mentioned in https://github.com/sinonjs/sinon/pull/1618.
> Updated for new and current release.

#### How to verify - mandatory
1. Check out this branch
2. `npm install`
3. cd `docs`
4. build docs: `bundle exec jekyll serve`
